lighthouse-core/report/html/renderer/element-screenshot-renderer.js

```diff
@@ -175,11 +182,11 @@ class ElementScreenshotRenderer {
       if (!screenshotElement) return;
 
       overlay.appendChild(screenshotElement);
-      overlay.addEventListener('click', () => {
+      containerEl.addEventListener('click', () => {
         overlay.remove();
       });
 
-      reportEl.appendChild(overlay);
+      containerEl.insertBefore(overlay, topbarEl.nextElementSibling);
```

Comment:
there seems to be significance to the fact that the overlap comes immediately after the topbar. if so, comment? if not, stick with `appendChild` on the container?

Reply:
ok

> if not, stick with appendChild on the container?

It doesn't render if placed after `lh-container`.